assets/javascripts/discourse/components/activity-pub-authorize.js

```diff
@@ -32,8 +107,15 @@ export default class ActivityPubAuthorize extends Component {
     this.verifiedDomain = false;
   }
 
+  @action
+  onDomainKeyDown(event) {
+    if (event.key === "Enter") {
+      this.verifyDomain();
```

Comment:
I think we need to also `preventDefault` here, when I test this in Safari, the page reloads while it verifies the domain (so user has to start over).

Reply:
Done: https://github.com/discourse/discourse-activity-pub/pull/133/commits/dd91bf2ebedd6f5805bd0482dc92bfca27742524